L1Trigger/L1THGCal/test/HGCalBackendStage1ParameterExtractor.cc

```diff
@@ -280,11 +282,15 @@ uint32_t HGCalBackendStage1ParameterExtractor::getTCaddress(int& tc_ueta, int& t
   }
 }
 
-uint32_t HGCalBackendStage1ParameterExtractor::getReducedModuleHash(int subdet, int layer, int ueta, int vphi) {
+uint32_t HGCalBackendStage1ParameterExtractor::getReducedModuleHash(HGCalTriggerModuleDetId moduleId) {
+
+  uint32_t subdetId = (uint32_t)(triggerTools_.isScintillator(moduleId));
+  uint32_t layer = triggerTools_.layerWithOffset(moduleId);
+  int moduleUEta = moduleId.moduleU(); //returns eta if scintillator
+  int moduleVPhi = moduleId.moduleV(); // returns phi if scintillator
```

Comment:
In principle these values are not negative in the sector 0, but could be worth testing that they are indeed positive, otherwise it would mess the hash values.

Reply:
Added a check that skips the TC if `moduleHash` is ill-defined